vhdbuilder/packer/install-dependencies.sh

```diff
@@ -472,15 +472,19 @@ PATCHED_HYPERKUBE_IMAGES="
 1.15.11-hotfix.20200529.1
 1.15.11-hotfix.20200714.1
 1.15.11-hotfix.20200817.1
+1.15.11-hotfix.20200824
```

Comment:
Just base image patched hyperkube for `hotfix.20200824`, can you add `.1` to all of `hotfix.20200824` images?